Dockerfile

```diff
@@ -36,17 +36,11 @@ ENV JAVA_HOME=/opt/jdk-mini
 ENV PATH="$PATH:$JAVA_HOME/bin"
 ENV DIRPATH /pitchfork
 
-
 # Create some dirs and copy pitchfork jar
 RUN mkdir -p $DIRPATH
 COPY target/pitchfork.jar $DIRPATH/
 
 RUN chmod 755 $DIRPATH/pitchfork.jar
```

Comment:
is this necessary? I think it should work without this line

Reply:
Yeap, not needed. Removed. Thanks :)